build/storage/tests/it/ptf_tests/tests/test_hot_plug_recipe.py

```diff
@@ -0,0 +1,73 @@
+# Copyright (C) 2022 Intel Corporation
```

Comment:
in the future this tests will be separate and open in new way, for now it's part of this test, first we have to check environment, if we create new tests it can be open in not proper order so for now it's the best way to prevent,
when we will have more tests we do this better

Reply:
changed